modules/gdnative/godot/dictionary.cpp

```diff
@@ -29,9 +29,10 @@
 /*************************************************************************/
 #include <godot/dictionary.h>
 
+#include "core/variant.h"
```

Comment:
Would have been good to put a comment, so that we don't delete the separating line in the future, thus breaking it again next time clang-format changes the include order.